src/clients/update_proposal.rb

```diff
@@ -428,7 +428,9 @@ def init_stuff
         # products to reselect after reset
         restore = []
 
-        Y2Packager::Resolvable.find(kind: :product, status: :selected).each do |product|
+        # preload the "transact_by" attribute to avoid querying libzypp again (see bsc#1176276)
+        Y2Packager::Resolvable.find({ kind: :product, status: :selected }, [:transact_by])
```

Comment:
@lslezak can you verify that it should go also for `master`?

Reply:
I already discussed it with @lslezak. Originally, it was a workaround (which is now properly fixed), but it is safe for `master`.